src/transformers/data/data_collator.py

```diff
@@ -166,8 +166,18 @@ class DataCollatorForTokenClassification:
     max_length: Optional[int] = None
     pad_to_multiple_of: Optional[int] = None
     label_pad_token_id: int = -100
-
-    def __call__(self, features):
+    return_tensors: str = 'pt'
+
```

Comment:
I would add a post init to check the selected framework in `return_tensors` is indeed available, to fail as early as possible if that's not the case.